datascience/tables.py

```diff
@@ -1247,20 +1260,25 @@ def sample(self, k=None, with_replacement=True, weights=None):
         original table.
 
         Args:
-            ``k`` -- specifies the number of rows (``int``)  to be sampled from
-                self. Default is k is equal to number of rows in self.
+            ``k`` -- specifies the number of rows (``int``) to be sampled from
+               the table. Default is k equal to number of rows in the table.
 
-            ``with_replacement`` -- (``boolean``), if true samples ``k`` rows
-                with replacement from self, else samples ``k`` rows without
-                replacement.
+            ``with_replacement`` -- (``bool``) By default TRUE; Samples ``k``
```

Comment:
Oops, I was unclear in my previous comment. I meant `True`, since that's how boolean values in Python are spelled.